### PR TITLE
Add support for `startOffset` and `endOffset` as expression

### PR DESCRIPTION
**Pull request by adimiz1**
### Brief Summary of Changes
Add support for `startOffset` and `endOffset` as expression

#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [x] New feature
- [ ] Bug fix
- [x] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

File: Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift [hunk 1]
```diff
@@ -142,6 +142,72 @@ class CLDTransformationExpressionsTests: BaseTestCase {
...
+        
+        let expectedResult = "ih_mul_2"
+        
+        // When
+        sut.setStartOffset(expression)
+        
+        let actualResult = sut.startOffset!
+        
+        // Then
+        XCTAssertFalse(actualResult.isEmpty, "x should stored new value")
```
Comment (const-cloudinary) [inline Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift:174]: Please fix error messages.

File: Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift [hunk 2]
```diff
@@ -142,6 +142,72 @@ class CLDTransformationExpressionsTests: BaseTestCase {
...
+        let actualResult = sut.startOffset!
+        
+        // Then
+        XCTAssertTrue(actualResult.isEmpty, "Empty expression should not be stored in params")
+    }
+    
+    func test_setStartOffset_inputExpression_shouldStoreNewValue() {
+        
+        // Given
+        let input       = "initialHeight * 2"
```
Comment (const-cloudinary) [inline Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift:163]: The value of the expression does not represent a real world use case, like `initialHeight` is not related to video offset... Please use more related variables, like `duration`. The test content is very confusing.

**Review by const-cloudinary — CHANGES_REQUESTED** [review]

File: Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift (hunk 2 above)
Comment (adimiz1) [reply Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift:163]: Fixed

File: Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift [hunk 3]
```diff
@@ -142,6 +142,72 @@ class CLDTransformationExpressionsTests: BaseTestCase {
...
+        
+        let expectedResult = "du_sub_3"
+        
+        // When
+        sut.setEndOffset(expression)
+        
+        let actualResult = sut.endOffset!
+        
+        // Then
+        XCTAssertFalse(actualResult.isEmpty, "actualy result should not be empty")
```
Comment (patrick-tolosa) [inline Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift:207]: Please fix the English error here :

`actualy result should not be empty` sounds odd (and has a typo)
while in the line below a space is missing and has a typo.

`acutalyResult and expectedResult should be eqaul`

**Review by patrick-tolosa — APPROVED** [review]

File: Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift (hunk 3 above)
Comment (adimiz1) [reply Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift]: Fixed

**Review by const-cloudinary — APPROVED** [review]

**Pull request merged**